hydra/video/107310631.jsonld

```diff
@@ -0,0 +1,40 @@
+{
+    "@context": [
+        "http://localhost:4000/hydra/@context/video.jsonld",
+        {
+            "@base": "http://localhost:4000"
+        }
+    ],
+    "@type": [
+        "http://videos.resftest.org/api/Video",
```

Comment:
Is this `Video` type defined anywhere? I didn't see it in the [video `@context` file](https://github.com/RESTFest/videos.restfest.org/pull/15/files#diff-bb4b097e9013cbcf878d9d2a7e6eb63d).

Reply:
It is defined in the [ApiDocumentation](https://github.com/RESTFest/videos.restfest.org/pull/15/files#diff-225c2b6b462a5df4a60313c6fbd53443R30)